branding/supplemental-ui/uyuni/uyuni-webui/js/vendor/langSelection.js

```diff
@@ -0,0 +1,11 @@
+function selectLanguage(langId){          
+
+    var str = window.location.pathname;
+    var regexPattern =/\/..\// ;
+
+    var matchCode = "/" + langId + "/" ;
+
+    var newURL = str.replace(regexPattern, matchCode);
+    window.location.href = newURL;
+
+}
```

Comment:
```suggestion
}

```
Git likes to have a newline at the end of each source file, so I would recommend adding it.